Formula/audacious.rb

```diff
@@ -0,0 +1,96 @@
+class Audacious < Formula
+  desc "Free and advanced audio player based on GTK+"
+  homepage "http://audacious-media-player.org"
+  url "http://distfiles.audacious-media-player.org/audacious-3.8.2.tar.bz2"
+  sha256 "bdf1471cce9becf9599c742c03bdf67a2b26d9101f7d865f900a74d57addbe93"
+  head "https://github.com/audacious-media-player/audacious.git"
+
+  option "with-app", "Build .app"
```

Comment:
True. Still, please remove this option. Quoting from our ["Acceptable Formulae" guidelines](http://docs.brew.sh/Acceptable-Formulae.html):

> Don’t make your formula build an .app (native macOS Application); we don’t want those things in Homebrew (but Homebrew Cask does). Make it build a command-line tool or a library.